From be42ad6a56fc9b39574b90b33b8b763d1bf5f7d8 Mon Sep 17 00:00:00 2001
From: Bruce Dawson <brucedawson@chromium.org>
Date: Tue, 8 Jun 2021 17:45:31 +0000
Subject: [PATCH] Render bindings with Python 3

Some of the bindings code-gen was moved to Python 3 but two build steps
were missed. This moves them over.

Bug: 1194277
Change-Id: I406ff8341dde51652bec94e59d9813814c7a17cf
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2946352
Reviewed-by: Kentaro Hara <haraken@chromium.org>
Commit-Queue: Bruce Dawson <brucedawson@chromium.org>
Cr-Commit-Position: refs/heads/master@{#890326}
---
 src/third_party/blink/renderer/bindings/BUILD.gn | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/third_party/blink/renderer/bindings/BUILD.gn b/src/third_party/blink/renderer/bindings/BUILD.gn
index b9f1c91051..d4cc47b5d1 100644
--- a/src/third_party/blink/renderer/bindings/BUILD.gn
+++ b/src/third_party/blink/renderer/bindings/BUILD.gn
@@ -65,9 +65,6 @@ if (use_blink_v8_binding_new_idl_interface) {
 
 template("collect_idl_files") {
   action_with_pydeps(target_name) {
-    # TODO(crbug.com/1194277): Investigate non-determinism in Py3 builds.
-    run_under_python2 = true
-
     script = "${bindings_scripts_dir}/collect_idl_files.py"
 
     inputs = invoker.inputs
@@ -138,9 +135,6 @@ collect_idl_files("web_idl_in_modules_for_testing") {
 }
 
 action_with_pydeps("web_idl_database") {
-  # TODO(crbug.com/1194277): Investigate non-determinism in Py3 builds.
-  run_under_python2 = true
-
   visibility += [ "//tools/privacy_budget/blink_apis:*" ]
 
   script = "${bindings_scripts_dir}/build_web_idl_database.py"
